openadapt/entrypoint.py

```diff
@@ -7,17 +7,21 @@
     multiprocessing.freeze_support()
 
 
-from openadapt.alembic.context_loader import load_alembic_context
-from openadapt.app import tray
 from openadapt.build_utils import redirect_stdout_stderr
-from openadapt.config import print_config
 
 
 def run_openadapt() -> None:
     """Run OpenAdapt."""
-    print_config()
-    load_alembic_context()
-    tray._run()
+    try:
+        from openadapt.alembic.context_loader import load_alembic_context
+        from openadapt.app import tray
+        from openadapt.config import print_config
+
+        print_config()
+        load_alembic_context()
+        tray._run()
+    except Exception:
```

Comment:
What do you think about logging the exception here? e.g.:

```
except Exception as exc:
    logger.warning(f"{exc=}")

Reply:
As discussed, let's add a call to `logger.warning` with comment explaining why this is here, a note that it won't print to stdout because of the redirection, and a TODO to log to a file (https://github.com/OpenAdaptAI/OpenAdapt/issues/734)

Reply:
Thank you. Can you please add a TODO to log all errors throughout the application?

Also what do you think about appending the date and time to the filename?

Reply:
Related: https://github.com/OpenAdaptAI/OpenAdapt/issues/734